test/playback.js

```diff
@@ -96,6 +96,16 @@ describe('A Google Music instance not playing music', function () {
   describe('playing music', function () {
     browserMusicUtils.playAnything();
     browserMusicUtils.waitForPlaybackStart();
+
+    browserUtils.execute(function getPlaybackDuration () {
+      return window.gmusic.playback.getSongDuration();
+    });
+
+    it('is a number great than 0', function () {
```

Comment:
This sentence doesn't make sense. The result would be "playing music is a number greater than 0". Try: `it('has a duration greater than 0')`

Also, we had a typo on great (should be greater)